### PR TITLE
[MM-20837] Add unit tests to the "ConfigGetCmd" mmctl command

### PR DESCRIPTION
**Pull request by artemigkh**
#### Summary
Added unit tests for the ConfigGetCmd mmctl command that covers the following test cases:
- Get a single nested config setting
- Get a block of config settings
- Assert Invalid Key error is returned when accessing nonexistent config field
- Assert error on client request is gracefully handled

#### Ticket Links
Jira: https://mattermost.atlassian.net/browse/MM-20837
Fixes mattermost/mattermost-server#13287

P.S - this repo didn't quite work for me out of the box, failing with the same errors CI is; I had to replace the import `"github.com/mattermost/mattermost-server/model"` in `license_test.go` with `"github.com/mattermost/mattermost-server/v5/model"` for it to build and have the tests run properly

### CODE REVIEW

**Comment by hanzei** [comment]
Hi @artemigkh ,

@mgdelacroix is OOO until the next year. Sorry for the delay in review.

File: commands/config_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+			},
+		}
+
+		mockErrorMessage := "Mock Internal Server Error"
+		mockError := &model.AppError{Message: mockErrorMessage}
+
+		s.client.
+			EXPECT().
+			GetConfig().
+			Return(mockConfig, &model.Response{Error: mockError}).
```
Comment (agnivade) [inline commands/config_test.go]: Let's return nil instead of `mockConfig` as I think that mocks the API more closely.

**Comment by artemigkh** [comment]
I think someone already wrote tests for this and merged it, don't think this PR is necessary anymore

**Comment by agnivade** [comment]
That seems right. @hanzei - should this be closed ?

**Comment by hanzei** [comment]
That is correct @agnivade. Sorry for the inconvenient @artemigkh and thank you for the time you spend on the PR.

**Pull request closed without merging**